test/txmeta-test.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+
+'use strict';
+
+const assert = require('./util/assert');
+const Network = require('../lib/protocol/network');
+const TXMeta = require('../lib/primitives/txmeta');
+
+const network = Network.get('regtest');
+
+
+describe('txmeta', function() {
+  it('should report correct number of tx confirmations', async () => {
```

Comment:
Let's have `JSON` in the test case name for clarity. Maybe `should return JSON for txmeta`.

Reply:
Ideally we should be testing that all JSON fields are what we expect, but we could limit to confirmations for this PR and fill those up later.